docs/ops/detection/ExperimentalDetectronPriorGridGenerator_6.md

```diff
@@ -0,0 +1,97 @@
+## ExperimentalDetectronPriorGridGenerator <a name="ExperimentalDetectronPriorGridGenerator"></a> {#openvino_docs_ops_detection_ExperimentalDetectronPriorGridGenerator_6}
+
+**Versioned name**: *ExperimentalDetectronPriorGridGenerator-6*
+
+**Category**: Object detection
+
+**Short description**: An operation *ExperimentalDetectronPriorGridGenerator* operation generates prior grids of specified sizes.
+
+**Attributes**:
+
+* *flatten*
+
+    * **Description**: *flatten* attribute specifies whether the output tensor should be 2D or 4D.
+    * **Range of values**:
+      * `true` - the output tensor should be 2D tensor
+      * `false` - the output tensor should be 4D tensor
+    * **Type**: boolean
+    * **Default value**: true
+    * **Required**: *yes*
+
+* *h* (*w*)
+
+    * **Description**: *h* (*w*) attribute specifies number of cells of the generated grid with respect to height (width).
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: 0
+    * **Required**: *yes*
+
+* *stride_x* (*stride_y*)
+
+    * **Description**: *stride_x* (*stride_y*) attribute specifies the step of generated grid with respect to x (y) coordinate.
+    * **Range of values**: non-negative float number
+    * **Type**: float
+    * **Default value**: 0.0
+    * **Required**: *yes*
```

Comment:
Could you write descriptions of `stride_x` and `stride_y` as separate descriptions? And more, these attributes are optional, see lines 80 and 81 of the file https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/vpu/graph_transformer/src/stages/exp_priorgridgenerator.cpp.

Reply:
Done.